SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied only to the most recent releases.
+
+## Reporting a Vulnerability
+
+To report a vulnerability, please [submit a new issue](https://github.com/CriminalInjuriesCompensationAuthority/cica-web/issues/new/choose).
+
+## Vulnerability Process
+
+1. Your report will be acknowledged within two business days.
+2. The team will investigate and update the issue with relevant information.
+3. If the team does not confirm the report, no further action will be taken and the issue will be closed.
+4. If the team confirms the report, the team will take action to fix it immediately:
+    1. Commits will be handled in a privately for review and testing.
+    2. Release a new patch version.
+    3. Write a wiki post disclosing the vulnerability.
```

Comment:
Not sure if we need this?
We could communicate the vulnerability fix with release notes?

Reply:
I'm happy to remove that bit. Was included in the interest of transparency, since these are the steps we'd follow if someone discovered a vulnerability we hadn't noticed, rather than just us reacting to audit failures.

Reply:
I think leaving it is fine and even having the wiki post as an option but I would keep the option to adding the vulnerability fix to the release notes only too.